spec/requests/api_keys_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "ApiKeys", type: :request do
+  describe "GET /api_keys" do
+    it "works! (now write some real specs)" do
```

Comment:
Yeah, either update the comment or remove the test.  I just don't want to checkin a TODO and wonder about it when we find it 2 months later.